app/cypress/support/commands.js

```diff
@@ -80,11 +80,23 @@ Cypress.Commands.add("clearMockedTime", () => {
   cy.clearCookie("mocks.mocked_timestamp");
 });
 
+Cypress.Commands.add(
+  "fillAndCheckNewProjectForm",
```

Comment:
Consider adding params for which funding stream and year to fill and check, just be a little more flexible for future use.

Reply:
I'm not sure what you mean--different params from what's on line 85 below?

Reply:
Nope... Sorry I was still shaking off vacation brain